build/configs/imxrt1020-evk/scripts/memory.ld

```diff
@@ -85,17 +85,25 @@
 
 MEMORY
 {
-  /* 64Mb of HyperFLASH */
+  /* 64MB of HyperFLASH */
```

Comment:
It has 8MB nor flash, not 64MB of hyperflash.

Reply:
Fixed.